client/static/js/app.js

```diff
@@ -1,14 +1,14 @@
 
 $(document).ready( function() {
-    var board = Board.init();
+    var Board = board.init();
```

Comment:
This is a nit-pick and not necessary for merging this, but for consistency, our object classes should be capitalized and local variables lowercase.  In Intellij you can do a `replaceAll` for `Board` to `board`, then change this line back to `Board` and the `board.js` to `Board`.